src/components/ChatBotMessages/index.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import ChatBot from "react-simple-chatbot";
+import { ThemeProvider } from "styled-components";
+import { useTranslation } from "react-i18next";
+import womanBot from "./images/woman.svg";
+import womanSeeksHelp from "./images/womanSeeksHelp.svg";
+import "./index.scss";
+import { Redirect } from "react-router-dom";
+
+const ChatBotMessages = () => {
+  const { t } = useTranslation();
+
+  // You can find chatbot documentation here https://lucasbassetti.com.br/react-simple-chatbot/#/
+  const theme = {
+    background: "#FFFEFC",
+    fontFamily: "Roboto",
+    headerBgColor: "#FFBFB5",
+    headerFontColor: "#fff",
+    headerFontSize: "15px",
+    botBubbleColor: "#C8D7C2",
+    botFontColor: "#fff",
+    userBubbleColor: "#FFBFB5",
+    userFontColor: "#fff",
+  };
+
+  const GoToSearhHelpPage = () => {
```

Comment:
typo

Reply:
since you are calling this a Page and naming it LikeThisForAComponent, we don't nest component declarations in other components. you dont have to put them in another file but move them all at the top of the file under the imports.

Reply:
and you redeclare the translation hook in each component